Microsoft.Toolkit.Diagnostics/Microsoft.Toolkit.Diagnostics.csproj

```diff
@@ -13,34 +13,35 @@
     </Description>
     <PackageTags>Windows;Community;Toolkit;WCT;UWP;Incremental;Loading;Collection;IncrementalLoadingCollection;String;Array;extensions;helpers</PackageTags>
   </PropertyGroup>
+
   <Choose>
-    <When Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
+    <When Condition="'$(TargetFramework)' == 'netstandard1.4'">
+      <!-- .NET Standard 1.4 doesn't have the Span<T> type, ValueTuple or the [Pure] attribute -->
       <ItemGroup>
-
-        <!-- .NET Standard 1.4 doesn't have the Span<T> type, ValueTuple or the [Pure] attribute -->
```

Comment:
I do agree on the same but only when the lines look congested. If they are easy on the eyes (_in this case_) then removing the un-necessary blank lines wouldn't affect readability.

This is also why I put multi-line comment's start and end tag on separate lines to avoid having to add additional new lines.

Reply:
I'm either way on the extra blank line, but especially with the ones below, the comments are about the packages, so it seems a bit odd to move them to outside the item group?

Reply:
The comment is about all 3 packages which are in the `ItemGroup`, not just `System.Diagnostics.Contracts`! That's why I moved them.